docs/teams/community_team/community_dashboard.rst

```diff
@@ -0,0 +1,17 @@
+Community dashboard
+###################
+
+The Community Dashboard uses the open-source :xref:`Savannah CRM` tool to import, analyze and visualize data from multiple sources in one central location.
+
+You can view the data from the last 90 days on the :xref:`Savannah Dashboard`.
+
+Currently the dashboad imports:
+
+-  All repositories listed under github.com/mautic
+-  Specified channels from mautic.slack.com
+-  All threads on forums.mautic.org
+-  All threads on reddit.com/r/mautic
+-  All Stack Overflow posts tagged with Mautic
+-  Specified meetup.com groups (coming soon)
```

Comment:
```suggestion
```

This is no longer relevant (we don't use meetup.com any more) so I would remove this line al together.